types/csp-hash-generator/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for csp-hash-generator 0.0
+// Project: https://github.com/miguel-a-calles-mba/csp-hash-generator
+// Definitions by: yb999 <https://github.com/bobcode99>
```

Comment:
the handle name is different

Reply:
Resolve https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58369/commits/050b96854a63859dafd7252f4a228ba0376da7a4